mne_bids/dig.py

```diff
@@ -451,18 +420,19 @@ def _read_dig_bids(electrodes_fpath, coordsystem_fpath,
         Filepath of the electrodes.tsv to read.
     coordsystem_fpath : str
         Filepath of the coordsystem.json to read.
-    raw : mne.io.Raw
-        The data as MNE-Python Raw object.
     datatype : str
         Type of the data recording. Can be ``meg``, ``eeg``,
         or ``ieeg``.
+    raw : mne.io.Raw | None
+        Optional. If passed in, should be the raw data as MNE-Python
+        Raw object. Will set montage read in via ``raw.set_montage(montage)``.
```

Comment:
see comment below, I think this can be made REQUIRED for now.

Reply:
It's not optional anyway, as it doesn't have a default value in the signature.